internal/tmpl/tmpl_test.go

```diff
@@ -152,6 +154,8 @@ func TestFuncMap(t *testing.T) {
 	var ctx = context.New(config.Project{
 		ProjectName: "proj",
 	})
+	wd, _ := os.Getwd()
```

Comment:
maybe add an error check? not sure how getwd can fail, but if it does, tests bellow will also fail and it could be a bit harder to debug...

```suggestion
	wd, err := os.Getwd()
	require.NoError(t, err)
```

Reply:
Indeed - good catch!